config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-
+  root :to => 'pages#homepage'
```

Comment:
We don't use hash rockets anymore :wink: that's old Ruby!

Please change into:

```ruby
root to: 'pages#homepage'
```

Also remove extra line beneath it, line 4.